products/extensions/b2b-components/shopping-lists/index.md

```diff
@@ -0,0 +1,10 @@
+---
+nav:
+  title: Shopping lists
+  position: 10
+
+---
+
+# Shopping lists
+
+The Shopping List application is designed to help users easily manage their shopping needs. It allows users to create, edit, and organize their shopping lists efficiently. With features such as item categorization, quantity tracking, and the ability to mark items as purchased, the Shopping List ensures that users can shop with ease and never forget essential items. Whether you are planning for a weekly grocery trip or a special event, the Shopping List app provides a simple and intuitive solution to keep track of all your shopping requirements.
```

Comment:
List with `s`. 

IMO `Shopping Lists  component` is better than the `Shopping Lists  application`